src/pages/signin/LoginForm.js

```diff
@@ -71,6 +92,13 @@ class LoginForm extends React.Component {
         }
     }
 
+    /**
+     * Clear Login from the state
+     */
+    clearLogin() {
+        this.setState({login: ''}, this.input.clear);
```

Comment:
Why call clear on this? Is `ExpensiTextInput` not a controlled component?

Reply:
Yeah. It's not. I just saw that we don't pass the value directly. `ExpensiTextInput` is using `value={this.value}`. Actually, we never faced a situation to make it controlled.

Reply:
Oh hmm weird I don't get it. We are passing a value - is it used or not ? 🤔 

It seems like a bad habit to call `this.input.clear` every time we update a value passed to `ExpensifyTextInput` to be an empty string. Why do we need to do this?

Reply:
I wasn't using it before but when after recent changes, `ExpensifyTextInput` value is not clearing on updating the state variable login. But as soon as you blur it, values clear. So I think that using `this.value` instead of the state variable is causing this issue on `ExpensifyTextInput`. 
As far as I know, it's not affecting other parts of the app so I don't think that refactor is necessary. If we face this issue again then I would suggest making the `ExpensifyTextInput` controlled.